test.js

```diff
@@ -12,92 +23,65 @@ const fixtures = [
 	'.dot.tmp'
 ];
 
-test.beforeEach(() => fixtures.forEach(fs.ensureFileSync));
-test.afterEach(() => fixtures.forEach(fs.removeSync));
+test.beforeEach(t => {
+	process.chdir(cwd);
+	t.context.tmp = tempfile();
+	fixtures.forEach(fixture => fs.ensureFileSync(path.join(t.context.tmp, fixture)));
+});
 
-test.serial('delete files - async', async t => {
+test('delete files - async', async t => {
+	process.chdir(t.context.tmp);
```

Comment:
Seemed like the only way to let this working (without options) together with line 27. If you don't care about testing it without options, we could remove all the `chdir` stuff.